internal/presenter/packages/spdx_json_presenter.go

```diff
@@ -113,3 +129,14 @@ func newSPDXJsonElements(catalog *pkg.Catalog) ([]spdx22.Package, []spdx22.File,
 
 	return packages, files, relationships
 }
+
+func cleanSPDXName(name string) string {
+	// remove # according to specification
+	name = strings.Replace(name, "#", "-", -1)
+
+	// remove : for url construction
+	name = strings.Replace(name, ":", "-", -1)
```

Comment:
is this needed?

Reply:
yea ":" is not in that non-reserved list so an input like `alpine:latest` would need to be stripped